### DISCUSSION TITLE
VS Code editor profile support

### DISCUSSION THREAD

**Discussion by crazy4pi314** [community/community#37162] [Codespaces]
VS Code has recently added [preview support](https://code.visualstudio.com/updates/v1_72#_settings-profiles) for profiles, allowing you to create different sets of user and default settings for the editor UI. As this feature is solidified, this would be an amazing thing to support in Codespaces (the menu seems to be there, but maybe not syncing the profiles). When you have a variety of projects that have large extensions that you don't want to load in a lighter weight Codespaces session, it would be great to as the Codespace is created to be able to select which profile to use.

**Comment by dmilburn** [comment]
Hi @crazy4pi314, hi from the Codespaces team. Thanks for the feature request! I've added an internal issue with this request for consideration though I can't guarantee when/if it'll be prioritized.
